evm/tools/test_builder/test_builder.py

```diff
@@ -1,3 +1,8 @@
+from typing import (  # noqa: F401
+    Dict,
+    List,
+    Any
+)
```

Comment:
There should be no blank lines between stdlib imports, and they should be ordered alphabetically